### ISSUE TITLE
Forbidden parameter name

### ISSUE THREAD

**Issue by BraisGabin** [detekt/detekt#7349]
## Expected Behavior of the rule
There are some functions where using the parameter name doesn't give any value it only makes de code more verbose. An example:

```kotlin
painterResource(id = R.drawable.ic_close)
```

The `id = ` provides no information.

This rule would be something similar to `ForbiddenImport`. We provide it but is the user of detekt that needs to list which functions doesn't allow paramenter name.

## Context
There are some functions on my code where I see this kind of pattern. The devs commit that type of code it because AS adds that `id = ` for them when they autocomplete `painterResource` function and they don't remove it.
